.github/workflows/crowdin-contributors.yml

```
@@ -0,0 +1,45 @@
+name: Crowdin Contributors Action
+
+on:
+  # When you push to the `main` branch
+  push:
+    branches: [ main ]
+  # And optionally, once every 12 hours
+  schedule:
+    - cron: '0 */12 * * *' # https://crontab.guru/#0_*/12_*_*_*
```

```suggestion
  schedule:
    - cron: '0 0 * * 1' # https://crontab.guru/#0_0_*_*_1
```

A suggestion to run this workflow weekly.

I don't think we want a new PR every time we push to main (would that end up creating a new PR every time we merge one?)

Expecting one of these to be reviewed every 12 hours seems ambitious.
I'm not sure how often we expect translation contributions to be made.

---

thanks, i can update it to run once a week

---

Done

---

:+1:

I think we should also remove the `push` trigger. I'm not 100% certain but the way I read it, every time we merge a PR a new crowdin PR will be (potentially) opened. I don't think we would want that.